src/WebSocketsServiceProvider.php

```diff
@@ -79,7 +79,7 @@ protected function registerRoutes()
 
     protected function registerDashboardGate()
     {
-        Gate::define('viewWebSocketsDashboard', function ($user = null) {
+        Gate::define('viewWebSocketsDashboard', function ($user) {
```

Comment:
The `$user` should still be `null` as a default. This way you can still access the dashboard locally without having the need of an authenticated user.